src/Samples/SubModelSeq.Views/SubModelSeq.Views.csproj

```diff
@@ -5,4 +5,10 @@
     <UseWpf>true</UseWpf>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Page Update="Counter.xaml">
+      <SubType>Designer</SubType>
+    </Page>
+  </ItemGroup>
+
```

Comment:
I think this can safely be removed.

Reply:
Yep.  Good catch.

For simplicity, I want to split off any changes that can be merged into `master` now.   The commit containing this change (which I will revert) is such a change.  I push such a commit to `master` and then rebase the branch in this PR.  I will mention here after that is done.

Reply:
Done.